includes/components/class-attribute.php

```diff
@@ -844,6 +847,11 @@ public function add_model_fields( $fields, $object ) {
 		foreach ( $attributes as $attribute_name => $attribute ) {
 			if ( ! isset( $fields[ $attribute_name ] ) ) {
 
+				// Check user attributes.
+				if ( 'user' === $model && ! ( $attribute['editable'] && $attribute['edit_field']['required'] && 'attachment_upload' !== $attribute['edit_field']['type'] ) ) {
```

Comment:
Let me know why this was required, it excludes attachment fields on the model level, this may affect other parts not just the registration form

Reply:
Probably we decided to exclude attachments from registration form because of it unstable behavior

Reply:
But you exclude it in registration form hook. Why is it needed to exclude in model field hook?